src/main/java/roomescape/reservation/infra/ReservationDaoImpl.java

```diff
@@ -35,14 +38,17 @@ public ReservationDaoImpl(JdbcTemplate jdbcTemplate) {
 
     @Override
     public Optional<Reservation> findById(Long id) {
-        String sql = "SELECT * FROM reservation WHERE id = ?";
+        String sql = "SELECT " +
+                "r.id as reservation_id," +
+                "r.name," +
+                "r.date," +
+                "t.id as time_id," +
+                "t.time as time_value " +
+                "FROM reservation r " +
+                "INNER JOIN reservation_time t ON r.time_id = t.id " +
+                "WHERE r.id = ?";
```

Comment:
```suggestion
        String sql = """
    SELECT 
        r.id AS reservation_id,
        r.name,
        r.date,
        t.id AS time_id,
        t.time AS time_value
    FROM reservation r
    INNER JOIN reservation_time t ON r.time_id = t.id
    WHERE r.id = ?
    """;
```
아래와 같이 관리하면 불필요한 + 단어를 제거할 수 있어 조금 더 가독성이 좋을 것 같습니다!

Reply:
하도 자바 8만 쓰다보니 멀티라인의 존재를 자꾸 잊게 되네요 ㅎㅎ
리마인드 감사드립니다! ㅎㅎ